blueprints/redirect-upload-requests/blueprint.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"meta": {
+		"title": "Pretty permalinks",
+		"description": "Set the permalink structure to use pretty permalinks.",
+		"author": "bgrgicak",
+		"categories": ["Settings"]
```

Comment:
Thank you @ivanblagdan! Would you please adjust the meta fields to explain the future readers what makes this Blueprint interesting? Thank you!